opendcs-rest-api/src/main/java/org/opendcs/odcsapi/dao/DbException.java

```diff
@@ -18,28 +18,28 @@
 public class DbException extends Exception
 {
 	private static final long serialVersionUID = 4552935582551955795L;
-	private String module = null;
-	private Exception cause = null;
-	
-	public DbException(String msg)
-	{
-		super(msg);
-	}
+	private final String module;
 	
 	public DbException(String module, Exception cause, String msg)
 	{
-		this(msg);
+		super(msg, cause);
 		this.module = module;
-		this.cause = cause;
 	}
 
-	public String getModule()
+	public DbException(Exception cause, String msg)
```

Comment:
1. This just looks weird being backwards.
2. cause should be `Throwable`

Reply:
agreed, but the other constructor uses the first String param for the "module" so I got stuck. I think this should just be a different exception class, especially since I think all DbExceptions are getting reported at error code 400 which doesn't seem right, at least across the board.